### ISSUE TITLE
How do I move the annoying [Browse] button?

### ISSUE THREAD

**Issue by pblanton** [Morningstar/GoASQ#7]
I feel stupid asking this, but this browse button and answers file name is terribly mis-placed.

![image](https://user-images.githubusercontent.com/1143649/50190056-93f6da80-02dc-11e9-9d21-7ec884cf5866.png)

I have searched the code and can't seem to find it anywhere. When I try to use the developer console in Chrome to isolate it, the console causes it to disappear. I'm sure it's part of a .js include from somewhere but I've had no luck tracking it down.

Any tips would be much appreciated.

**Comment by mendickxiao** [comment]
Which browser do you use? We will use the chrome/firefox. I don't meet this
issue. You can also debug in your machine, and see what happen it. Welcome
to be a contributor.

Phillip H. Blanton <notifications@github.com> 于2018年12月19日周三 上午7:52写道：

**Comment by pblanton** [comment]
Thank you. I'm using Firefox in Linux. I'll try Chrome too to see if I get the same issue.

**Comment by pblanton** [comment]
Good catch. It's not there in Chrome. It seems to be a Firefox-only issue.

Update: After a restart it is in chrome too.

**Comment by pblanton** [comment]
Apparently is is left over from VSAQ. Looking at the VSAQ demo, (https://vsaq-demo.withgoogle.com/vsaq.html?qpath=questionnaires/webapp.json) you can see that this was the only interface provided on the VSAQ questionnaire. The GoASQ questionnaire has a nice control bar at the bottom with lots more functionality and I presume the old webkit [Browse] button from VSAQ was de-comissioned, but for some reason but is still hanging around in Firefox. I'm trying to narrow it down.

**Comment by yippibrian** [comment]
I am not sure why this is browser specific, but the old VSAQ Browse button should not show up.  I believe this button shows up by including a file browser widget, probably via something like the following code:

<input type="file"  ... >

I notice that this is used in the src/vsaq/static/questionnaire/templates.soy file.  Check that out.

**Comment by pblanton** [comment]
Thanks for the tip. I made it go away by editing line 88 of vsaq.html and commenting out the following line...

`<input type="file" id="answer_file" />`

It doesn't seem to have harmed the application other than making the [Browse] button go away. 

Here's the comment in context...

`            <div class="box-container">
              <div class="box">
                <a class="maia-button eh-submissions js-submissions-modal-trigger" id="_vsaq_all_submissions" data-submissions="submissions">GoASQ TOC</a>
                <label class="pk-import"><a class="maia-button">Import</a></label>
                <a class="maia-button" id="_vsaq_export_questionnaire">Export</a>
                <a class="maia-button eh-clear" id="_vsaq_clear_questionnaire">Reset</a>
                <a class="maia-button eh-save" id="_vsaq_save_questionnaire">Save</a>
              </div>
`
